simplq/src/store/selectedQueue/selectedQueueSlice.js

```diff
@@ -48,6 +48,8 @@ export default selectedQueueSlice.reducer;
 
 export const selectQueueName = (state) => state.selectedQueue.queueName;
 
+export const selectRemoveTokens = (state) => state.selectedQueue.removedTokens;
```

Comment:
selectRemovedTokens here and other places

Reply:
As per the screenshot that I have attached, we can see that in removedTokens List we are not getting the timestamp of the new removed item, for this I am maintaining the queueHistory and moreover if we refresh we don't have anything in the queueHistory so we can't just add the first item that was newly added  which we could have done in all other cases.

Reply:
Resolved in latest Comment